plugins/data-management/src/AddTrackWidget/components/PasteConfigWorkflow.tsx

```diff
@@ -51,10 +55,15 @@ const PasteConfigAddTrackWorkflow = observer(function ({
             const session = getSession(model)
             const conf = JSON.parse(val)
             const confs = Array.isArray(conf) ? conf : [conf]
-            confs.forEach(c => session.addTrackConf(c))
-            confs.forEach(c => c.trackId)
-            model.clearData()
-            session.hideWidget(model)
+            if (
+              isSessionWithAddTracks(session) &&
+              isSessionModelWithWidgets(session)
+            ) {
+              confs.forEach(c => session.addTrackConf(c))
+              confs.forEach(c => model.view.showTrack(c.trackId))
```

Comment:
there was a bug here where I think it wanted to showTrack(c.trackId) but instead it was just forEach c=>c.trackId